### ISSUE TITLE
Notification not working, noifys after opening the app

### ISSUE THREAD

**Issue by azumukupoe** [TwidereProject/Twidere-Android#1075]

**Comment by Famicoman** [comment]
+1 This is what I came here to post. 

I'd expect to get notifications for directs and mentions while not actively using the app. Instead, only after I open the app do I get notifications in my task bar, some of which could be hours or days old by the time I see them.

I have confirmed that notifications are enabled for each account and that my refresh interval is 10 minutes. Running Android 5.0.1.

**Comment by Nikoh72** [comment]
yeah the notifications come in late even when within the app and you refresh the notifications still does not show up

**Comment by nghuyy** [comment]
After 3 years, this bug doen't fixed :(
